tests/tox.ini

```diff
@@ -0,0 +1,11 @@
+[tox]
+envlist = py35,py36,py37
```

Comment:
I don't think we can use py35 here since we utilize f-strings which were introduced in 3.6.

Reply:
I can remove it. Honestly the python version here isn't all that terribly important. If we only want ``ocs-ci`` to support ``py36`` or ``py37`` we can pick one and then name the tox environment something more descriptive than the python version. That's probably be a good idea anyway.

Reply:
Yeah that sounds like a better approach. If we have multiple envs in `envlist` won't we end up executing tests once per env? I don't think that's what we want to happen for our ocs tests.

Reply:
I've renamed the env ``ocs-tests`` and defaulted to ``py37``. I'm open to suggestions on better naming for sure, but I think that works. You can specify which env you want to run with the ``-e`` flag so I don't think that'd be a problem.